core/src/components/chip/chip.scss

```diff
@@ -104,13 +101,14 @@
   @include margin(-4px, -4px, -4px, 8px);
 }
 
-
 // Chip Avatar
 // ---------------------------------------------
 
 ::slotted(ion-avatar) {
-  width: 24px;
-  height: 24px;
+  flex-shrink: 0;
```

Comment:
Why was this added?

Reply:
This was part of the fix for that (pre-existing) bug that I discovered while working this ticket. I added the info for that to the PR description. Without this line, the avatars look like this if the chip width is narrower than its contents:

<img width="221" alt="Screenshot 2023-08-02 at 5 02 35 PM" src="https://github.com/ionic-team/ionic-framework/assets/14926794/c715aac9-2559-4304-8507-0f4d1f1eb39b">